test/unit/test_LIALogicMkTerms.cc

```diff
@@ -21,132 +22,132 @@ class LIALogicMkTermsTest: public ::testing::Test {
 TEST_F(LIALogicMkTermsTest, testIsNumTerm) {
     PTRef c = logic.mkBoolVar("c");
     ASSERT_FALSE(logic.isNumTerm(c));
-    PTRef t = logic.mkNumVar("t");
+    PTRef t = logic.mkIntVar("t");
     ASSERT_TRUE(logic.isNumTerm(t));
-    PTRef e = logic.mkNumTimes(logic.mkNumVar("e"), logic.mkConst(-114));
+    PTRef e = logic.mkTimes(vec<PTRef>{logic.mkIntVar("e"), logic.mkIntConst(-114)});
```

Comment:
```suggestion
    PTRef e = logic.mkTimes(logic.mkIntVar("e"), logic.mkIntConst(-114));
```
The `vec` is not needed, no?